assets/locales/service.cy.toml

```diff
@@ -307,3 +307,7 @@ one = "There are too many variables within this dataset"
 [MaximumVariablesErrorDescription]
 description = "Description for a maximum variable exceeded error"
 one = "Remove a variable to continue"
+
+[MaxCellsErrorPanelDescription]
+description = "Description for warning panel if maximum cells exceeded error"
+one = "<strong>This dataset has more than one million cells, the maximum number permitted.</strong>"
```

Comment:
Are you using `<strong>` for **bold** or for the screenreader? Not sure this is correct depending on how you're using it.

Reply:
Might be wrong both ways actually... believe a warning message should have a `role="alert"` for a screenreader and for **bold** there should be a `font-weight`.

Reply:
`<strong> ` is for visual effect only in this context - I think if I make sure the panel picks up `role="alert"` the semantic requirements will be fulfilled